### PR TITLE
Fix issue 19939 - std.format count width error

### PR DESCRIPTION
**Pull request by shove70**

### CODE REVIEW

File: std/format.d [hunk 1]
```diff
@@ -2739,6 +2739,19 @@ if (is(FloatingPointTypeOf!T) && !is(T == enum) && !hasToString!(T, Char))
     formatTest( S2(2.25), "S" );
 }
 
+// bugzilla 19939
+@system unittest
```
Comment (thewilsonator) [inline std/format.d]: Please make this `@safe`

**Comment by thewilsonator** [comment]
Probably want to target stable here, this is pretty obviously not going to break anything.

Otherwise Looks good. Thanks!

File: std/format.d (hunk 1 above)
Comment (shove70) [reply std/format.d]: OK, it has been revised.

**Comment by shove70** [comment]
> Probably want to target stable here, this is pretty obviously not going to break anything.
> 
> Otherwise Looks good. Thanks!

Sorry, I made a mistake. Let me mention a PR again.

**Comment by ZombineDev** [comment]
@shove70 you don't need close your pull requests everytime you want to update something. Simply use git to clone the repository locally, checkout an existing branch, or make a new one, make the changes, commit and push. When you have more changes commit and push again. When you need to rebase - rebase and force push.

If you need more assistance, I'd happy to help.

**Pull request closed without merging**